src/test/scala/ai/privado/languageEngine/csharp/tagger/collection/CollectionTaggerTest.scala

```diff
@@ -0,0 +1,52 @@
+package ai.privado.languageEngine.csharp.tagger.collection
+
+import ai.privado.languageEngine.csharp.CSharpTestBase
+import ai.privado.model.{CatLevelOne, Constants, SourceCodeModel}
+import io.shiftleft.semanticcpg.language.*
+
+class CollectionTaggerTest extends CSharpTestBase {
+
+  "Types with Route annotations in code" should {
+    "be tagged as part of Collection tagger" in {
+      val (cpg, _) = code(
+        List(
+          SourceCodeModel(
+            """
+              |using System;
+              |
+              |namespace Foo {
+              |
+              |  [Route("api/[controller]")]
+              |  [ApiController]
+              |  public class SomeController : ControllerBase
+              |  {
+              |    [HttpGet("{email}")]
+              |    public IActionResult GetEmail(string email)
+              |    {
+              |       return ControllerContext.MyDisplayRouteInfo(email);
+              |    }
+              |
+              |    [HttpGet("[action]/{firstName}")]
+              |    public IActionResult Print(string firstName)
+              |    {
+              |       return ControllerContext.MyDisplayRouteInfo(firstName);
+              |    }
+              |  }
+              |}
+              |""".stripMargin,
+            "Test.cs"
+          )
+        )
+      )
+
+      val List(emailMethod) = cpg.method.nameExact("GetEmail").l
```

Comment:
Can we cover more test cases, Ex- different ways to pass the routes in annotation, what if route present on class not on method etc